raiden/ui/web/src/app/components/open-dialog/open-dialog.component.html

```diff
@@ -24,7 +24,13 @@ <h1 mat-dialog-title>Open Channel</h1>
                formControlName="settle_timeout"
                required min="1"/>
```

Comment:
Please, set the `min` value here too. For UI/UX in general, when possible, it's always better to **prevent** the user from doing an action which will lead to an error (e.g. entering and invalid amount) than showing an error. The error message and validation and in the dialog can stay, but as a fallback measure. Saw this pattern in other places too (e.g. decimals in amount input, etc), but we may look into fixing them in the future

Reply:
@andrevmatos Please if you see stuff I missed please point out :)